microhapdb/tests/test_cli.py

```diff
@@ -644,3 +644,34 @@ def test_cli_summarize(capsys):
 """
     print(observed)
     assert observed.strip() == expected.strip()
+
+
+def test_cli_fasta_locus_multimarker(capsys):
+    arglist = [
+        "marker",
+        "mh14SHY-003.v1",
+        "mh14SHY-003.v2",
+        "mh14SHY-003.v3",
+        "mh14SHY-003.v4",
+        "mh02KK-134.v1",
+        "mh02KK-134.v2",
+        "mh02KK-134.v3",
+        "mh02KK-134.v4",
+        "--format=fasta",
+    ]
+    args = get_parser().parse_args(arglist)
+    microhapdb.cli.main(args)
+    terminal = capsys.readouterr()
+    observed = terminal.out
+    expected = """
+>mh02KK-134 GRCh38:chr2:160222879-160223013 mh02KK-134.v3=20,44,59 mh02KK-134.v1=20,44,59,123 mh02KK-134.v2=20,44,59,65,107,123 mh02KK-134.v4=20,44,59,65,72,87,107,123
+TACCCTTGGCAGGAACCCTCACTACCTAAGGATGGGCAATGGCTTATGAGTGAGAAACACGGAGCCGTGGGAACTCAGAA
+TGACATGCTACCTGGAGATTGTGGTAACGCCCTGTTTTTTTGTGGGCATATCTA
+>mh14SHY-003 GRCh38:chr14:57983921-57984213 mh14SHY-003.v1=10,14,16,26,102,108,109,161,192,199 mh14SHY-003.v4=10,14,102,108,199,262,281 mh14SHY-003.v3=102,108,199 mh14SHY-003.v2=102,108,199,262,281
+GTAGGAGTGATGTACGGGGCACCTACTTGGGGTTCACATGCTGGCCCCTTTATTGAGTTCATTCTGAATCCAGAAGCTTG
+GCAGAGTTCAGCCAGATGGCAGGGTGAGCGCCCTGCCTTCCTGGTAGTCTCTTCTTCTGCAAGGGAATAGGAGGCGTTCA
+CCCTCCTTTGTTCAAGAGTCTATTTCTAGGGGCCTATCAGCCCAGGGTCCCTTCTCCAGCTTTCTCAGGAGGCCCCACAT
+CATCAGGCAATTAGCTCTCTAGTGGGTATAACTGCTACTGCCACAACCACTG
```

Comment:
This new test captures what the new output looks like for loci with multiple marker definitions.